src/getCoinCombination.test.js

```diff
@@ -1,10 +1,33 @@
 'use strict';
 
+
 describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+  it('should be declared', () => {
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it('should return [1, 0, 0, 0] for 1 cent', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
 
-  it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+  it('should return [1, 1, 0, 0] for 6 cents', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
+
+  it('should return [2, 1, 1, 0] for 17 cents', () => {
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+  });
+
+  it('should return [0, 0, 0, 2] for 50 cents', () => {
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+  });
+
```

Comment:
Test case for negative inputs is missing. It's important to test how your function behaves with unexpected inputs.